manifests/generated/vmi-resource.yaml

```diff
@@ -7,6 +7,19 @@ metadata:
     kubevirt.io: ""
   name: virtualmachineinstances.kubevirt.io
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .metadata.creationTimestamp
+    name: Age
+    type: date
+  - JSONPath: .status.phase
+    name: Phase
+    type: string
+  - JSONPath: .status.interfaces[0].ipAddress
```

Comment:
I can understand that for kubevirt-developers and deoming the IP is handy. Since the IP changes between restarts and since there are all the other DNS and `virtctl/kubectl` ways which allow you to connect to your VMI in an IP independent way, I really think that this gives the wrong message to users.

Reply:
Assuming position 0 will only work if  only the pod network is present.